src/main/java/mil/dds/anet/InitializationCommand.java

```diff
@@ -34,7 +34,7 @@ public void configure(Subparser subparser) {
         .help("set administrative position name");
     subparser.addArgument("--adminFullName").action(Arguments.store()).required(true)
         .help("set administrator's full name; use format: LASTNAME, Firstname");
-    subparser.addArgument("--adminDomainName").action(Arguments.store()).required(true)
+    subparser.addArgument("--adminDomainUserName").action(Arguments.store()).required(true)
```

Comment:
I'd use `adminDomainUsername` (no capital N) to be consistent with the database field name (change in this file and `build.gradle`).